### PR TITLE
feat(words): add words

### PR DESCRIPTION
**Pull request by HansRobo**
| word | |
| --- | --- |
| interpolable | https://ejje.weblio.jp/content/interpolable |
| publically | https://ejje.weblio.jp/content/publically | 
| collinearity | https://ejje.weblio.jp/content/collinearity | 
| postprocessor | https://ejje.weblio.jp/content/postprocessor |

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -781,6 +782,7 @@
     "inpaints",
     "inplace",
     "insertafter",
+    "interpolable",
```
Comment (kenji-miyake) [inline .cspell.json:785]: There is an alternative form. How about replacing it?
https://ejje.weblio.jp/content/interpolatable

File: .cspell.json [hunk 2]
```diff
@@ -1226,6 +1229,7 @@
     "ptpd",
     "ptree",
     "ptrs",
+    "publically",
```
Comment (kenji-miyake) [inline .cspell.json:1232]: Maybe can be replaced by `publicly`?
https://en.wiktionary.org/wiki/publically

**Review by kenji-miyake — CHANGES_REQUESTED** [review]

**Comment by HansRobo** [comment]
@kenji-miyake Thank you for suggesting alternative form of words!
At first, I'll try to replate the words to them in code and check if cspell occurs no error about them.
After that, I'll determine whether delete these words or edit to alternative form of these words.

File: .cspell.json (hunk 1 above)
Comment (HansRobo) [reply .cspell.json:785]:
```suggestion
    "interpolatable",
```

File: .cspell.json (hunk 2 above)
Comment (HansRobo) [reply .cspell.json:1232]: `publicly` doesn't occur cspell error.
So, I'll delete the word from `.cspell.json`
```suggestion
```

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**